raiden/raiden_service.py

```diff
@@ -476,6 +479,12 @@ def register_registry(self, registry_address):
             self.manager_token[manager_address] = token_address
             self.channelgraphs[token_address] = graph
 
+            self.tokens_connectionmanagers[token_address] = ConnectionManager(
+                self,
```

Comment:
Yeah, thinking about it, the ConnectionManager could become a property of the App -- ideally it should only be acting as a consumer of the programmatic API and should probably not live in the same branch of the dependency tree.
```
# like this
App --- RaidenService --- RaidenAPI
    \      | (event callbacks)   ^
     \     v                     | api calls (via reference to RaidenAPI)
      \_ ConnectionManager ------

# instead of this
         (event callbacks & circular reference)
                       |
App --- RaidenService <-> ConnectionManager
            \_ RaidenAPI  <_api_calls_/  (via reference to RaidenService)
```

haha, not sure if my little graph is actually helpful :)